backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -51,7 +50,7 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
       .set("spark.memory.offHeap.size", "2g")
       .set("spark.unsafe.exceptionOnMemoryLeak", "true")
       .set("spark.sql.autoBroadcastJoinThreshold", "-1")
-      .set("spark.sql.sources.useV1SourceList", "avro")
+      .set("spark.sql.sources.useV1SourceList", "avro,parquet")
```

Comment:
cc @PHILO-HE  I'm not sure why are we using v2 to read parquet. I changed to use v1 if it does not break your original idea.